scripts/EA_adler2007.json

```diff
@@ -0,0 +1,34 @@
+{
+    "citation": "Peter B. Adler, William R. Tyburczy, and William K. Lauenroth. 2007. Long-term mapped quadrats from Kansas prairie:demographic information for herbaceaous plants. Ecology 88:2673.",
+    "description": "Demographic data for testing current theories in plant ecology and forecasting the effects of global change.",
+    "name": "Kansas plant quadrats (Ecological Archives 2007)",
+    "shortname": "Adler2007",
+    "tables": {
+        "monthly_ppt": {
+            "cleanup": "Cleanup(correct_invalid_value, nulls=['NA'])",
+            "replace_columns": "[('jan', 'january'), ('feb', 'february'), ('mar', 'march'), ('apr', 'april'), ('jun', 'june'), ('jul', 'july'), ('aug', 'august'), ('sep', 'september'), ('oct', 'october'), ('nov', 'november'), ('dec', 'december')]"
```

Comment:
yah I am thinking` "[('jan', 'january'), ('feb', 'february')]",` I have not yet tested or looked at this,

I suggest that you test both and see what you get. Also a reminder, as you work on this part, try to keep the tests that you have used, we could include some of them as test cases later.